components/partials/common/trade/advanced-settings/index.vue

```diff
@@ -241,6 +211,10 @@ export default Vue.extend({
       }
     },
 
+    handleHasErrorUpdateEvent(hasError: boolean): void {
+      this.$emit('update-has-advanced-settings-errors', hasError)
+    },
+
     handleReduceOnlyCheckboxToggle(checked: boolean) {
       this.$emit('set-reduce-only', checked)
```

Comment:
Let's follow the pattern of `action:item` for the event names, here lets say we can have `update:reduceOnly` and above we have have `set:advancedSettingsErrors`